aspnetcore/tutorials/first-mvc-app/working-with-sql.md

```diff
@@ -90,35 +101,40 @@ Replace the contents of *Program.cs* with the following code:
 
 [!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/Program.cs)]
 
-Test the app
+Test the app.
 
 # [Visual Studio](#tab/visual-studio)
 
-* Delete all the records in the DB. You can do this with the delete links in the browser or from SSOX.
-* Force the app to initialize (call the methods in the `Startup` class) so the seed method runs. To force initialization, IIS Express must be stopped and restarted. You can do this with any of the following approaches:
+Delete all the records in the database. The data can be deleted by one of the following approaches:
 
-  * Right click the IIS Express system tray icon in the notification area and tap **Exit** or **Stop Site**
+* Selecting all the `Delete` links in the browser.
+* Selecting the database in SQL Server Object Explorer and ...
```

Comment:
This is all wrong. Selecting SSOX has nothing to do with init. Revert with minor cleanup. We don't use ellipsis this way.